### DISCUSSION TITLE
What is the national sport of 🇧🇹?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#719] [Galaxy Trivia] [answered]
*answer with emoji*

**Comment by peterbaccaro** [comment, marked as answer]
🏹 Archery is the national sport in Bhutan.

**Comment by SANKARAMDAS** [comment]
🏹 Sport in Bhutan comprises both traditional Bhutanese and modern international games.  🏹Archery is the national sport in Bhutan.

**Comment by thuongtruong1009** [comment]
The national sport of 🇧🇹 Bhutan is archery. 🏹
